src/zc/zrs/nagios.py

```diff
@@ -0,0 +1,187 @@
+from __future__ import print_function
+##############################################################################
+#
+# Copyright (c) 2015 Zope Corporation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+
+import optparse
+import json
+import re
+import socket
+import struct
+import sys
+import ZODB.TimeStamp
+
+def connect(addr):
+    if isinstance(addr, tuple):
```

Comment:
Based on discussion with @jimfulton, we're going to ignore this concern until we're ready to revamp the IPv6 support in a more rational way.